.gitignore

```diff
@@ -83,3 +83,6 @@ sample.tex
 
 # pytest related data file for slow tests
 .ci/durations.log
+
+test_lambdify.py
```

Comment:
Why has this been added in the gitignore?

Reply:
It is a mistake. I just wanted to do it locally.